citi-onboarding/Aqua-Jr
# Feature/expertise-area

## What I did
- Created the expertise area component
- Made it responsive

## How to test
1. Access the branch through your terminal:
```
   git checkout develop
   git pull
   git checkout feature/expertise-area
```
2. Access the server folder and run the server:
```
   cd /server
   nodemon index.js
```

3. Access the client folder in another terminal window and start React:
```
   cd /client
   yarn start
```
4. In your browser, go to:
``` 
   localhost:3000
```
5. Check if it is responsive (from 300px to 3000px)
6. Check if something is different from the mockup in both versions
7. Check if the categories buttons are filtering the services
8. Try to add a area with a different category
9. Check if the Slick is working properly
    9.1. If both arrows are working in both versions
    9.2. If the dots are changing colors based on the slider
    9.3. If the dots are clickable
    9.3. if the Main slide is more visible than the others (only in the computer version)
